go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/spotify/flyte-flink-plugin
 go 1.16
 
 require (
-	github.com/envoyproxy/protoc-gen-validate v0.1.0
-	github.com/flyteorg/flyteidl v0.18.25
-	github.com/flyteorg/flyteplugins v0.5.40
-	github.com/flyteorg/flytestdlib v0.3.15
-	github.com/golang/protobuf v1.4.3
+	github.com/envoyproxy/protoc-gen-validate v0.6.1
+	github.com/flyteorg/flyteidl v0.18.50
+	github.com/flyteorg/flyteplugins v0.5.50
+	github.com/flyteorg/flytestdlib v0.3.23
```

Comment:
we seem to have a regression on flytestdlib where maps are not being merged as expected